src/ServiceControl.Config/UI/Shell/VersionCheckerHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace ServiceControl.Config.UI.Shell
+{
+    using Newtonsoft.Json;
+    using ServiceControlInstaller.Engine.Configuration.ServiceControl;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public static class VersionCheckerHelper
+    {
+        public static async Task<Release> GetDownloadUrlForNextVersionToUpdate(string currentVersion)
+        {
+            Version current = new Version(currentVersion);
+            List<Release> releases = await GetVersionInformation().ConfigureAwait(false);
+
+            Release topversion = releases.Select(t => (t.Version, t)).Max().t;
```

Comment:
I did not know that VaueTuple's sorted that way. That's neat :)

The other way to do it would have been `.SortByDescending(t => t.Version).First()`. That may be less efficient under the covers if it has to sort the whole list.